itests/hcatalog-unit/src/test/java/org/apache/hive/hcatalog/listener/TestDbNotificationListener.java

```diff
@@ -211,6 +203,11 @@ public void onAlterTable (AlterTableEvent tableEvent) throws MetaException {
       pushEventId(EventType.ALTER_TABLE, tableEvent);
     }
 
+    @Override
+    public void onAlterDatabase(AlterDatabaseEvent dbEvent) throws MetaException {
+      pushEventId(EventType.ALTER_DATABASE, dbEvent);
+    }
+
```

Comment:
Why is this introduced?

Reply:
Mock class needs to push ALTER_DATABASE event in order to get recorded